### DISCUSSION TITLE
Call a Route to Resource Edit Page from User Menu?

### DISCUSSION THREAD

**Discussion by korematter** [filamentphp/filament#5936] [Help] [answered]
Hi 

I know resource pages are tightly coupled but wondering if it is possible to call a route to a resource edit page from the UserMenuItem class and perhaps pass in a model record as the parameter? Trying not to recreate another Edit page with a custom Edit page if I don't have to.

AppServiceProvider.php
```
Filament::serving(function () {
                UserMenuItem::make()
                    ->label('Studio')
                    ->url(route( Resource Edit Page, 'record' => auth()->user()->modelRecord  )
                    ->icon('heroicon-s-cog'),
            ]);
        });
```

Thanks!
Steve

**Comment by danharrin** [comment, marked as answer]
`ResourceName::getUrl('edit', ['record' => auth()->user()])`

**Comment by korematter** [comment]
Thanks Dan! That worked perfectly! Learning something new every day! Cheers!
